src/libs/ReportUtils.js

```diff
@@ -873,7 +884,8 @@ function getDisplayNameForParticipant(login, shouldUseShortForm = false) {
 function getDisplayNamesWithTooltips(participants, isMultipleParticipantReport) {
     return _.map(participants, (participant) => {
         const displayName = getDisplayNameForParticipant(participant.login, isMultipleParticipantReport);
-        const tooltip = participant.login ? Str.removeSMSDomain(participant.login) : '';
+        const avatar = UserUtils.getDefaultAvatar(participant.login);
```

Comment:
I think this caused a regression where we are unable to load a user's uploaded/custom avatar in some cases as identified on https://github.com/Expensify/App/issues/20683

I believe I've got a fix so will push it up and request reviews from ya'll.

Reply:
PR is here https://github.com/Expensify/App/pull/20727